Dockerfile

```diff
@@ -1,14 +1,5 @@
-FROM node:stretch-slim
-
-RUN apt update && apt install -y git openssh-server ca-certificates openssl jq gettext xmlstarlet build-essential
-
-RUN npm install sfdx-cli prettier prettier-plugin-apex --global
+FROM codelicia/apex-prettier:latest
+RUN npm install --global sfdx-cli
 RUN sfdx --version
```

Comment:
I'd suggest merging this command with the previous `RUN`. Having an extra cache layer with this seems unnecessary to me 😬

Reply:
(not to mention you will save even more re$ource$)

Reply:
That is a good point! --> .

Thank you! ❤️